packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+  --token-tooltip-padding-horizontal: 12px;
+  --token-tooltip-padding-vertical: 8px;
+
+  // Radius:
+  --token-tooltip-border-radius: 5px;
+
+  // Animation:
+  --token-tooltip-transition-function: cubic-bezier(0.54, 1.5, 0.38, 1.11);
+
+  // Focus highlight position:
+  --token-tooltip-tab-focus-offset: -2px;
+
+  // Colors:
+  --token-tooltip-color-background-primary: var(--token-color-palette-neutral-700);
+  --token-tooltip-color-foreground-primary: var(--token-color-foreground-high-contrast); // not sure on best naming for this token
```

Comment:
Are these names ok for the colors or should we reference "high-contrast" or use some other naming?

(These colors are for the dark background of the tooltip "bubble" and the white text inside it.)

Reply:
Sorry I updated the names now.